matching-web/src/main/java/com/graphhopper/matching/http/MatchServlet.java

```diff
@@ -64,6 +64,7 @@
     public void doPost(HttpServletRequest httpReq, HttpServletResponse httpRes)
             throws ServletException, IOException {
 
+    	logger.info("posted");
```

Comment:
@karussell, @kodonnell: Should we remove this or change to logger.debug (change was done by @kodonnell)?